requirements.txt

```diff
@@ -90,15 +94,17 @@ QtPy==1.9.0
 requests==2.24.0
 requests-oauthlib==1.3.0
 rsa==4.6
-scikit-learn==0.23.2
+scikit-learn==0.22.2
 scipy==1.5.2
 Send2Trash==1.5.0
 Shapely==1.7.1
 six==1.15.0
+sklearn==0.0
```

Comment:
This is just an alias for scikit-learn, which is only in place for backwards compatibility.